app/controllers/v0/open_api_controller.rb

```diff
@@ -0,0 +1,21 @@
+# frozen_string_literal: true
+
+module V0
+  class OpenApiController < ApplicationController
+    service_tag 'platform-base'
+    skip_before_action :authenticate
+
+    def index
+      openapi_file_path = Rails.public_path.join('openapi.json')
+
+      if File.exist?(openapi_file_path)
+        openapi_spec = JSON.parse(File.read(openapi_file_path))
+        openapi_spec['servers'] =
```

Comment:
<img width="294" height="109" alt="Screenshot 2025-10-22 at 3 34 51 PM" src="https://github.com/user-attachments/assets/99e3c6ec-98cc-45d3-bb69-fb1be45fc235" />

^ Adds this